Gigs/Gigs/Application/GigController.swift

```diff
@@ -192,19 +192,9 @@ class GigController {
                 return
             }
             
-            let encoder = JSONEncoder()
-            encoder.dateEncodingStrategy = .iso8601
-            
-            do {
-                let newGig = try encoder.encode(gig)
-                request.httpBody = newGig
-            } catch {
-                completion(.failure(.encodingError))
-            }
-            
-    }.resume()
-        
+            self.gigs.append(gig)
```

Comment:
We don't want to append to gigs until we know that the post request was successful